sound/hybrid_font.h

```diff
@@ -120,7 +130,49 @@ class HybridFont : public SaberBase {
       PlayPolyphonic(effect);
     }
   }
+  void StartSwing() override {
+    if (!guess_monophonic_) {
+      if (!swing_player_) {
+        swing_player_ = PlayPolyphonic(&swng);
+      } else {
+        second_swing_player_ = PlayPolyphonic(&swng);
+      }
+    } else {
+      PlayMonophonic(&swing, &hum);
+    }
+  }
+
+  float SetSwingVolume(float swing_strength, float mixhum) {
+    if (IsSwingPlaying()) {
+      float accent_volume = powf(swing_strength, config_.ProffieOSSwingVolumeSharpness) * config_.ProffieOSMaxSwingVolume;
```

Comment:
ProffieOSMaxSwingVolume seems to be misnamed, as it is not a maximum.

Reply:
its the maximum multiple for the swing sound over the hum, much like the smooth swing pairs.